runregistry-rest/rest.py

```diff
@@ -1,237 +1,21 @@
-__author__ = "Roland Sipos"
-__credits__ = [""]
-__version__ = "0.0.8"
-__maintainer__ = "Roland Sipos"
-__email__ = "roland.sipos@cern.ch"
-
-import os, io, gzip, tarfile
-import flask
-import sys
-
-from flask import Flask, render_template, request, redirect, url_for, send_file
-
-from flask_restful import Api, Resource
-from flask_httpauth import HTTPBasicAuth
-from flask_redis import FlaskRedis
-from flask_caching import Cache
-
-from authentication import auth
-
-postgres = False
-
-if "-p" in sys.argv or os.environ.get("RGDB", None):
-    import backends.pg_queries as queries
-    import backends.pg_backend as db
-
-    postgres = True
-else:
-    import backends.ora_queries as queries
-    import backends.ora_backend as db
-
-"""
-Specs
-"""
-# Create an APISpec
-# spec = APISpec(
-#    title='Swagger Sensors',
-#    version='1.0.0',
-#    openapi_version='2.0',
-#    plugins=[
-#        FlaskPlugin(),
-#        MarshmallowPlugin(),
-#    ],
-# )
-
+#!/usr/bin/env python3
 """
 Main app
-"""
-app = Flask(__name__)
-app.config["MAX_CONTENT_LENGTH"] = 32 * 1000 * 1000
-app.config["UPLOAD_EXTENSIONS"] = [".gz", ".tgz"]
-app.config["UPLOAD_PATH"] = "uploads"
-app.config["CACHE_TYPE"] = "simple"
-cache = Cache(app)
-api = Api(app)
 
+To control set the following environment variables:
+DATABASE_URI - URI for sqlalchemy to use
+DEBUG - set to TRUE for flask debugging
+RUN_START - set to your starting run number, does not need to be changed after first launching as run number will be iterated from this initial value
```

Comment:
I'm not showing `RUN_START` used by any of the code in runregistry....

Reply:
I thought that it would automatically get the env from daq-kube

Reply:
I think this comment goes in the other service....